docs/user-tutorial/benchmarks/micro-benchmarks.md

```diff
@@ -60,11 +60,40 @@ Large scale matmul operation using `torch.matmul` with one GPU.
 
 ### `cublas-function`
 
-TODO
+#### Introduction
+
+The common cublas functions with parameters in models (resnet, vgg, densenet, bert, gpt2, lstm) traning.
+
+Cublas functions support:
+ - cublasSgemm
+ - cublasSgemmStridedBatched
+ - cublasGemmStridedBatchedEx
+ - cublasGemmEx
+ - cublasCgemm3mStridedBatched
+ - cublasCgemm
+
+#### Metrics
+
+| Name                                                     | Unit      | Description                                                       |
+|----------------------------------------------------------|-----------|-------------------------------------------------------------------|
+| cublas-function/name_${function_name}_${parameters}_time | time (us) | The mean time to execute the cublas function with the parameters. |
 
 ### `cudnn-function`
 
-TODO
+#### Introduction
+
+The common cudnn functions with parameters in models (resnet, vgg, densenet, bert, gpt2, lstm) traning.
+
+Cudnn functions support:
```

Comment:
The support functions for cuDNN are as follows:

Reply:
Updated